synapse/handlers/typing.py

```diff
@@ -185,7 +185,7 @@ def _push_update(self, member, typing):
 
     async def _push_remote(self, member, typing):
         try:
-            users = await self.state.get_current_users_in_room(member.room_id)
+            users = await self.state.get_users_in_room(member.room_id)
```

Comment:
Should this be `self.store` instead of `self.state`?

Reply:
sigh, here I thought was a PR I couldn't mess up...